### PR TITLE
Switched to using new MongoDB service in Bluemix

### LINKED ISSUE #1029: Update Bluemix Deployment Instructions To Use Different Mongo Service
The MongoLab service in Bluemix will be going away at the end of November.  We need to update the deployment instructions as well as the auto deployment button to use a different Mongo service.

### PR DESCRIPTION
**Pull request by ryanjbaxter**
Fixes #1029 
The MongoLabs service in Bluemix is being removed so we need to use a different Mongo service.

### CODE REVIEW

File: config/env/cloud-foundry.js [hunk 1]
```diff
@@ -91,4 +93,4 @@ module.exports = {
       }
     }
   }
-};
+};
```
Comment (ilanbiala) [inline config/env/cloud-foundry.js]: newline

File: config/env/cloud-foundry.js [hunk 2]
```diff
@@ -3,7 +3,9 @@
 var cfenv = require('cfenv'),
   appEnv = cfenv.getAppEnv(),
   cfMongoUrl = appEnv.getService('mean-mongo') ?
-  appEnv.getService('mean-mongo').credentials.uri : undefined;
+  appEnv.getService('mean-mongo').credentials.uri ?
```
Comment (ilanbiala) [inline config/env/cloud-foundry.js]: This looks too complicated

Comment (ryanjbaxter) [inline config/env/cloud-foundry.js]: @ilanbiala would you prefer I broke it up into a traditional if else so it is clearer?

Comment (ilanbiala) [inline config/env/cloud-foundry.js]: Probably

File: README.md [hunk 3]
```diff
@@ -196,17 +196,18 @@ We welcome pull requests from the community! Just be sure to read the [contribut
 Cloud Foundry is an open source platform-as-a-service (PaaS).  The MEANJS project
 can easily be deployed to any Cloud Foundry instance.  The easiest way to deploy the
 MEANJS project to Cloud Foundry is to use a public hosted instance.  The two most popular
-instances are [Pivotal Web Services](https://run.pivotal.io/) and 
+instances are [Pivotal Web Services](https://run.pivotal.io/) and
 [IBM Bluemix](https://bluemix.net).  Both provide free trials and support pay-as-you-go models
-for hosting applications in the cloud.  After you have an account follow the below steps to 
+for hosting applications in the cloud.  After you have an account follow the below steps to
```
Comment (ilanbiala) [inline README.md]: Why newline here?

File: config/env/cloud-foundry.js [hunk 4]
```diff
@@ -1,9 +1,16 @@
 'use strict';
 
 var cfenv = require('cfenv'),
-  appEnv = cfenv.getAppEnv(),
-  cfMongoUrl = appEnv.getService('mean-mongo') ?
-  appEnv.getService('mean-mongo').credentials.uri : undefined;
+  appEnv = cfenv.getAppEnv();
+var cfMongoUrl = undefined;
+if(appEnv.getService('mean-mongo')) {
```
Comment (ilanbiala) [inline config/env/cloud-foundry.js]: space

File: config/env/cloud-foundry.js [hunk 5]
```diff
@@ -1,9 +1,16 @@
...
+  var mongoCreds = appEnv.getService('mean-mongo').credentials;
+  if(mongoCreds.uri) {
```
Comment (ilanbiala) [inline config/env/cloud-foundry.js]: space

Comment (ilanbiala) [inline config/env/cloud-foundry.js]: This can be a ternay, just having 2 was confusing

File: README.md (hunk 3 above)
Comment (ryanjbaxter) [inline README.md]: I dont think i added a new line I just deleted a space at the end of the lines

Comment (ilanbiala) [inline README.md]: Okay, can you remove it?

File: config/env/cloud-foundry.js (hunk 1 above)
Comment (ilanbiala) [inline config/env/cloud-foundry.js]: newline

File: README.md (hunk 3 above)
Comment (ryanjbaxter) [inline README.md]: i'm confused you want me to add the trailing spaces back in?  Why?

Comment (ilanbiala) [inline README.md]: Make lines 201 and 202 one line.

File: README.md [hunk 6]
```diff
@@ -196,17 +196,18 @@ We welcome pull requests from the community! Just be sure to read the [contribut
 Cloud Foundry is an open source platform-as-a-service (PaaS).  The MEANJS project
```
Comment (ilanbiala) [inline README.md]: This section all the way up the bullets should actually be on 1 line.

**Comment by mleanos** [comment]
LGTM

**Comment by ilanbiala** [comment]
@lirantal LGTM.

**Comment by codydaig** [comment]
LGTM

**Pull request merged**